tests/chainer_tests/distributions_tests/test_kldivergence.py

```diff
@@ -38,6 +38,12 @@ def encode_params(self, params, is_gpu=False):
 
         return params
 
+    def make_beta_dist(self, is_gpu=False):
+        a = numpy.random.uniform(1, 10, self.shape).astype(numpy.float32)
+        b = numpy.random.uniform(1, 10, self.shape).astype(numpy.float32)
```

Comment:
I understood your intension.  It's a rounding-off error near 1.  How about testing params >= 0.5?  BTW, it seems `0.1` is big enough to avoid sampling 0.
```
>>> (stats.beta(0.5, 0.5).rvs(100000000) == 1).sum()
0
>>> (stats.beta(0.3, 0.3).rvs(100000000) == 1).sum()
841
>>> (stats.beta(0.1, 0.1).rvs(100000000) == 0).sum()
0
```